app/build.gradle

```diff
@@ -8,8 +8,8 @@ android {
         applicationId "io.pslab"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 15
-        versionName "2.0.14"
+        versionCode 16
+        versionName "2.0.15"
```

Comment:
Send a PR with only this change. Discard the rest.

Reply:
So should I make a PR to development branch?